ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/services/SandboxSubmissionService.scala

```diff
@@ -92,7 +93,13 @@ class SandboxSubmissionService private (
         recordOnLedger(commands).transform {
           case Failure(error) =>
             logger.warn(s"Submission of command ${commands.commandId.unwrap} has failed.", error)
-            Failure(error)
+            // quick work-around to handle back-pressure coming from Akka
```

Comment:
@francesco-da it turns out that pushing this down to the core requires more work as it affects heart-beats, the time-service etc. They will need to handle the case of back-pressure. Do you still insist on keeping this in the scope of this PR?

Reply:
I would merge this as soon as I get a green CI and address it right after in a separate one. See #423